docs/content/docs/custom-resource/pod-template.md

```diff
@@ -53,13 +53,13 @@ spec:
   flinkVersion: v1_16
   flinkConfiguration:
     taskmanager.numberOfTaskSlots: "2"
+  serviceAccount: flink
   podTemplate:
     apiVersion: v1
     kind: Pod
     metadata:
       name: pod-template
     spec:
-      serviceAccount: flink
```

Comment:
serviceAccount is part of the spec, please dont change this

Reply:
Once its there it fails upon validation.

https://github.com/apache/flink-kubernetes-operator/blob/main/flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/validation/DefaultValidator.java#L523

I had to move it to get the deployment to work.

Reply:
ah sorry, I looked at your change backwards, this is correct :)